tools/initRepo/.vscode/launch.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
Would rather not get rid of these either.
When we come to NPM this, that'll probably be the time.